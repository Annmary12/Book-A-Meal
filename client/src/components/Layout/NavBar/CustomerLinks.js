import React, { Fragment } from 'react';
import classNames from 'classnames';
import { Link } from 'react-router-dom';
import styles from './navbar.scss';

const CustomerLinks = ({ pathname }) => (
  <Fragment>
    <li>
      <Link
        className={classNames({
          [styles.active]: pathname === '/menus'
        })}
        to="/menus"
      >
        Menus
      </Link>
    </li>
    <li>
      <Link
        className={classNames({
          [styles.active]: pathname === '/orders'
        })}
        to="/orders"
      >
        Order history
      </Link>
    </li>
  </Fragment>
);

export default CustomerLinks;
