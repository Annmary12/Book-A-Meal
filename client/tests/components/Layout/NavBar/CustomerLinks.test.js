import React from 'react';
import CustomerLinks from '../../../../src/components/Layout/NavBar/CustomerLinks';

describe('CustomerLinks component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<CustomerLinks />);

    expect(wrapper).toMatchSnapshot();
  })
})