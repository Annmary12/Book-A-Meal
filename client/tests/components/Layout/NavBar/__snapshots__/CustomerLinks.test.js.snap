// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomerLinks component should match snapshot 1`] = `
<Fragment>
  <li>
    <PageLink
      text="Menus"
      to="/menus"
    />
  </li>
  <li>
    <PageLink
      text="Order history"
      to="/orders"
    />
  </li>
</Fragment>
`;
